simulation/traffic_simulator/include/traffic_simulator/traffic_lights/v2i_traffic_light_publisher.hpp

```diff
@@ -20,39 +20,39 @@
 #include <rclcpp/rclcpp.hpp>
 #include <string>
 #include <traffic_simulator/hdmap_utils/hdmap_utils.hpp>
-#include <traffic_simulator/traffic_lights/traffic_light_manager_base.hpp>
+#include <traffic_simulator/traffic_lights/configurable_rate_updater.hpp>
 
 namespace traffic_simulator
 {
+class TrafficLightManager;
+
 template <typename Message>
-class V2ITrafficLightManager : public TrafficLightManagerBase
+class V2ITrafficLightManager : public ConfigurableRateUpdater
```

Comment:
At the design stage, this class was called `TrafficLightPublisher`, but did you forget to change the name?

Reply:
Yes, that is correct. I missed the renaming of this class. I will do that :+1: 
Architecture, however, names this class as `V2ITrafficLightPublisher` since different traffic light publishers might have different implementations. Is this name okay, or would you prefer `TrafficLightPublisher`?

Reply:
Oh, sorry!
`V2ITrafficLightPublisher` would be fine!

Reply:
I renamed the class! Thank you! :+1: